### ISSUE TITLE
Major workaround for sudo

### ISSUE THREAD

**Issue by SidneySmi** [Trevor1134/AdminFun#3]
I've noticed that even if a person isn't able to have /sudo used on them, if a player has access to /adminfun playerchat they can just add a slash and make it force a command ex: /adminfun playerchat Admin /kickall

**Comment by KingFaris10** [comment]
If they don't have /sudo could you not deny them the permission node to use playerchat too...?

**Comment by SidneySmi** [comment]
Yes but for my server, I don't want my moderators to sudo people though I'm fine with them talking as other people. Furthermore regardless of what you can do, this is an issue.

**Comment by KingFaris10** [comment]
Don't know if this project is still being worked on
